### ISSUE TITLE
add a way for people to chat with one another like irc

### ISSUE THREAD

**Issue by lostfile1** [andrewwyatt/photonbbs#7]
maybe add some command or some thing that lets people just message one another in a channel instead of having to use /w or /r and some offline mail box system maybe

**Comment by andrewwyatt** [comment]
I'm not sure what you're asking for here as far as messaging.  If two people are in the same channel and they type a message and press enter, the message is sent to everyone in the channel.  If two people want to talk only to each other, they can create a private channel.  Could you describe what you're asking for in more detail?

An offline message box like memoserv is a pretty good idea.  I'm not sure when I'll get to implementing it though.

**Comment by lostfile1** [comment]
yeah some thing like that

**Comment by y4my4my4m** [comment]
@lostfile1 I think what @andrewwyatt is trying to say is that it already works this way

**Comment by andrewwyatt** [comment]
Yes, exactly.  I started to look at adding a memoserv like functionality, but I never completed it.
